fuzzers/binary_only/qemu_launcher/src/harness.rs

```diff
@@ -98,7 +98,7 @@ impl Harness {
         }
         let len = len as GuestReg;
 
-        unsafe { self.qemu.write_mem(self.input_addr, buf) };
+        self.qemu.write_mem(self.input_addr, buf).unwrap();
```

Comment:
`.map_err(|e| Error::unknown(format!("Failed to write memory to {:?}: {e:?}", self.input_addr)))?;`

But anyway qemu doesn't seem to return an error?

Reply:
Ah you changed it. Nice!